_episodes/06-func.md

```diff
@@ -528,7 +543,16 @@ help(numpy.loadtxt)
 ~~~
 Help on function loadtxt in module numpy.lib.npyio:
 
-loadtxt(fname, dtype=<class 'float'>, comments='#', delimiter=None, converters=None, skiprows=0, usecols=None, unpack=False, ndmin=0)
+loadtxt(fname, 
+        dtype=<class 'float'>, 
+        comments='#', 
+        delimiter=None, 
+        converters=None, 
+        skiprows=0, 
+        usecols=None, 
+        unpack=False, 
+        ndmin=0)
+
```

Comment:
Let's not break lines here. We'll have to update our scripts not to complain about long lines here.